rest-api/genomic/genomic_biobank_menifest_handler.py

```diff
@@ -0,0 +1,58 @@
+"""
+Create a genomic biobank manifest CSV file and uploads to biobank samples bucket subfolders.
+"""
+
+import clock
+import config
+import pytz
+from offline.sql_exporter import SqlExporter
+from code_constants import GENOME_TYPE
+from config import GENOMIC_BIOBANK_MANIFEST_FOLDER_NAME
+
+_US_CENTRAL = pytz.timezone('US/Central')
+_UTC = pytz.utc
+OUTPUT_CSV_TIME_FORMAT = '%Y-%m-%d-%H-%M-%S'
+_MANIFEST_FILE_NAME_ARRAY_PREFIX = 'Genomic-Manifest-AoU_Array'
+_MANIFEST_FILE_NAME_WGS_PREFIX = 'Genomic-Manifest-AoU_WGS'
+
+def create_and_upload_genomic_biobank_manifest_file(genomic_set_id, genome_type, timestamp=None):
+  result_filename = _get_output_manifest_file_name(genomic_set_id, genome_type, timestamp)
+  bucket_name = config.getSetting(config.BIOBANK_SAMPLES_BUCKET_NAME)
```

Comment:
our standard biobank samples upload is in /reconciliation and the Biobank uploads to the root folder. Is this the behavior you want ?

Reply:
Yes, this sub folder is by requirement.

Reply:
They want to use the same biobank bucket but different sub folder for this genomic test.